### PR TITLE
enable PWA for student app (#332)(#333)

### LINKED ISSUE #332: Zbadanie tematu Progressive Web App i opisanie czy i czego nam brakuje czego żeby je wdrożyć
Jeśli 💩 => poszukać alternatyw/rozwiązań

### LINKED ISSUE #333: Wdrożenie Progressive Web App do aplikacji studenta

### PR DESCRIPTION
**Pull request by ostatni5**
enable PWA for student
split public folder

### CODE REVIEW

**Review by michal367 — CHANGES_REQUESTED** [review]
make public_dir to be loaded as 3rd arg (`process.argv[3]`) and build_dir be 4th
make public_dir like src_dir
in the end build command should work like this:
`react-app-rewired build src/lecturer public/lecturer build/lecturer`
(don't forget about another commands like start)

`set PUBLIC_URL=/lecturer` may not work on heroku (windows instruction)
instead of using PUBLIC_URL var just use name of folder in index.html, for example: 
`/student/manifest.json`

delete all those comments from index.html

app on firefox and chrome have different names. one browser gets name from manifest.json, second browser gets from title
long name of app in manifest.json - barely fit on screen
obraz

File: public/student/robots.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
```
Comment (michal367) [inline public/student/robots.txt:1]: remove

Comment (ostatni5) [reply public/student/robots.txt:1]: i dont see why i should

**Review by michal367 — APPROVED** [review]

File: public/student/index.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="/student/favicon.ico" />
```
Comment (michal367) [inline public/student/index.html:6]:
```suggestion
    <link rel="icon" href="/student/favicon.ico" />
    <!-- Favicon Author 
    © 2016 - 2021 PNG All.
    https://creativecommons.org/licenses/by-nc/4.0/
    http://www.pngall.com/lecture-png/download/42561 -->
```

File: public/lecturer/index.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="/lecturer/favicon.ico" />
```
Comment (michal367) [inline public/lecturer/index.html:6]:
```suggestion
    <link rel="icon" href="/lecturer/favicon.ico" />
    <!-- Favicon Author 
    © 2016 - 2021 PNG All.
    https://creativecommons.org/licenses/by-nc/4.0/
    http://www.pngall.com/lecture-png/download/42561 -->
```

**Review by rozchlastywacz — APPROVED** [review]
10/10

**Pull request merged**